docs/source/docker/getting_started/first_steps.rst

```diff
@@ -271,8 +271,27 @@ The command schedules a job with the following configurations:
 For more details and options regarding the worker config, head to :ref:`docker-configuration`.
 For more details and options regarding the selection config, head to :ref:`worker-selection`.
 
+Monitoring the compute worker run
```

Comment:
Use capitals for the words as we did for the rest of the docs
`Monitoring the compute worker run` --> `Monitoring the Compute Worker Run`

<img width="285" alt="image" src="https://user-images.githubusercontent.com/1522766/190457725-6c5f55d0-d7f1-4c7f-9083-7f2ae2fec210.png">